docs/source/guide/FAQ.md

```diff
@@ -93,5 +93,18 @@ If you find that after annotating audio data, the visible audio wave doesn't mat
 ffmpeg -y -i audio.mp3 -ar 8k -ac 1 audio.wav
 ```
 
+## HTML label offsets are in the wrong places
```

Comment:
Unfortunately, this is not the only reason for incorrect offsets.. It might be caused by complex CSS and some tricky html tags.. @hlomzik correct me please if I'm wrong OR add these cases here

Reply:
Or maybe it's better to disable this minification completely.

Reply:
pinging @niklub to comment on the potential of disabling minification

Reply:
I added a few sentences to acknowledge that this isn't the only cause, but the most common one